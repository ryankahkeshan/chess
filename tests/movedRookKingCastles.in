game human human
move e2 e4
move e7 e5
move g1 f3
move g8 f6
move f1 c4
move f8 c5
move h1 g1
move h8 f8
move g1 h1
move f8 h8
move e1 g1
move e1 e2
move e8 g8
