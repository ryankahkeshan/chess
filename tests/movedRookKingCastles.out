8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2 PPPPPPPP
1 RNBQKBNR

  abcdefgh

8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ _P_ _
3 _ _ _ _ 
2 PPPP PPP
1 RNBQKBNR

  abcdefgh

8 rnbqkbnr
7 pppp_ppp
6  _ _ _ _
5 _ _ p _ 
4  _ _P_ _
3 _ _ _ _ 
2 PPPP PPP
1 RNBQKBNR

  abcdefgh

8 rnbqkbnr
7 pppp_ppp
6  _ _ _ _
5 _ _ p _ 
4  _ _P_ _
3 _ _ _N_ 
2 PPPP PPP
1 RNBQKB_R

  abcdefgh

8 rnbqkb r
7 pppp_ppp
6  _ _ n _
5 _ _ p _ 
4  _ _P_ _
3 _ _ _N_ 
2 PPPP PPP
1 RNBQKB_R

  abcdefgh

8 rnbqkb r
7 pppp_ppp
6  _ _ n _
5 _ _ p _ 
4  _B_P_ _
3 _ _ _N_ 
2 PPPP PPP
1 RNBQK _R

  abcdefgh

8 rnbqk_ r
7 pppp_ppp
6  _ _ n _
5 _ b p _ 
4  _B_P_ _
3 _ _ _N_ 
2 PPPP PPP
1 RNBQK _R

  abcdefgh

8 rnbqk_ r
7 pppp_ppp
6  _ _ n _
5 _ b p _ 
4  _B_P_ _
3 _ _ _N_ 
2 PPPP PPP
1 RNBQK R 

  abcdefgh

8 rnbqkr _
7 pppp_ppp
6  _ _ n _
5 _ b p _ 
4  _B_P_ _
3 _ _ _N_ 
2 PPPP PPP
1 RNBQK R 

  abcdefgh

8 rnbqkr _
7 pppp_ppp
6  _ _ n _
5 _ b p _ 
4  _B_P_ _
3 _ _ _N_ 
2 PPPP PPP
1 RNBQK _R

  abcdefgh

8 rnbqk_ r
7 pppp_ppp
6  _ _ n _
5 _ b p _ 
4  _B_P_ _
3 _ _ _N_ 
2 PPPP PPP
1 RNBQK _R

  abcdefgh

Invalid move
8 rnbqk_ r
7 pppp_ppp
6  _ _ n _
5 _ b p _ 
4  _B_P_ _
3 _ _ _N_ 
2 PPPPKPPP
1 RNBQ_ _R

  abcdefgh

Invalid move
Final Score:
White: 0
Black: 0
